runtime/tests/checker/reference_test.go

```diff
@@ -790,10 +790,8 @@ func TestCheckReferenceIndexingIfReferencedIndexable(t *testing.T) {
           fun test() {
               let rs <- [<-create R()]
               let ref = &rs as &[R]
-              var other <- create R()
-              ref[0] <-> other
```

Comment:
Do we still have tests for the non-reference / owned case that swaps? Same below

Reply:
Yes, they are in `swap_test.go` file.